src/main/scala/chiseltest/coverage/Coverage.scala

```diff
@@ -0,0 +1,20 @@
+package chiseltest.coverage
+
+import firrtl._
+import firrtl.annotations.{Annotation, NoTargetAnnotation}
+
+/** Coverage counts returned from the simulator interface. */
+case class TestCoverage(counts: List[(String, Long)]) extends NoTargetAnnotation
```

Comment:
Can you define (in a comment) what the strings mean?

Reply:
Will do.